Tests/BaseObjectTests.swift

```diff
@@ -16,181 +18,183 @@ class BaseObjectTests: QuickSpec {
 
-                    expect(callbackWasCalled) == true
+                        expect(callbackWasCalled) == true
 
-                    callbackWasCalled = false
-                    contextObject.trigger(eventName)
+                        callbackWasCalled = false
+                        anotherBaseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
+                        expect(callbackWasCalled) == false
+                    }
                 }
-            }
 
-            describe("off") {
-                it("Callback should not be called if removed") {
-                    let listenId = baseObject.on(eventName, callback: callback)
-                    baseObject.off(listenId)
-                    baseObject.trigger(eventName)
+                describe("off") {
+                    it("doesn't executes callback function if it is removed") {
+                        let listenId = baseObject.on(eventName, callback: callback)
+                        baseObject.off(listenId)
+                        baseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
-                }
-                it("Callback should not be called if removed, but the others should") {
-                    var anotherCallbackWasCalled = false
-                    let anotherCallback: EventCallback = { _ in
-                        anotherCallbackWasCalled = true
+                        expect(callbackWasCalled) == false
                     }
+                    it("doesn't execute callback if it is removed, but the others are called") {
+                        var anotherCallbackWasCalled = false
+                        let anotherCallback: EventCallback = { _ in
+                            anotherCallbackWasCalled = true
+                        }
 
-                    let listenId = baseObject.on(eventName, callback: callback)
-                    baseObject.on(eventName, callback: anotherCallback)
+                        let listenId = baseObject.on(eventName, callback: callback)
+                        baseObject.on(eventName, callback: anotherCallback)
 
-                    baseObject.off(listenId)
-                    baseObject.trigger(eventName)
+                        baseObject.off(listenId)
+                        baseObject.trigger(eventName)
 
-                    expect(callbackWasCalled) == false
-                    expect(anotherCallbackWasCalled) == true
+                        expect(callbackWasCalled) == false
+                        expect(anotherCallbackWasCalled) == true
+                    }
                 }
-            }
-
-            describe("stopListening") {
-                it("Should cancel all event handlers") {
-                    baseObject.on(eventName, callback: callback)
-                    baseObject.on("another-event", callback: callback)
 
-                    baseObject.stopListening()
+                describe("stopListening") {
```

Comment:
According to the Project conventions, we should add the # prefix on the Method Name.

``context("#stopListening")``